content/dart/concepts/type-conversion/type-conversion.md

```diff
@@ -0,0 +1,496 @@
+  var c = (2351.44464661).toStringAsExponential();
+  var d = 254645466111.toStringAsExponential(3);
+  var e = 1224.toStringAsExponential(20);
+  var f = 5.25.toStringAsExponential(0);
+
+  print(a);
+  print(b);
+  print(c);
+  print(d);
+  print(e);
+  print(f);
+
+}
+```
+
+The `Shell` output:
+
+```Shell
+1.000e+0
+2.565e+5
+2.35144464661e+3
+2.546e+11
+1.22400000000000000000e+3
+5e+0
+```
+
+---
+
+## toStringAsPrecision() Method
+
+## toStringAsPrecision Implementation
+
+```pseudo
+String toStringAsPrecision(int precision);
+```
+
+The `toStringAsPrecision` method converts the input number to a `double` and returns a string representation of that value with exactly `precision` significant digits.
+
+The parameter `precision` must be an integer satisfying:
+  
+  1 <= `precision` <= 21.
+
+## Example1
+
+```dart
+void main() {
+
+  var a = 1.toStringAsPrecision(3);
+  var b = (256463.51554546).toStringAsPrecision(1);
+  var c = (2351.44464661).toStringAsPrecision(2);
+  var d = 254645466111.toStringAsPrecision(3);
+  var e = 1224.toStringAsPrecision(20);
+  var f = 5.25.toStringAsPrecision(12);
+
+  print(a);
+  print(b);
+  print(c);
+  print(d);
+  print(e);
+  print(f);
+
+}
+```
+
+The `Shell` output:
+
+```Shell
+1.00
+3e+5
+2.4e+3
+2.55e+11
+1224.0000000000000000
+5.25000000000
+```
+
+---
+
+## toDouble() Method
+
+The `toDouble()` method converts a value to a double.
+
+## toDouble Method Example
+
+```dart
+void main() {
+  int intValue = 42;
+  double doubleValue = intValue.toDouble();
+
+  var checkCondition = (doubleValue == 42.0) ? true : false;
+
+  print(checkCondition);
+}
+```
+
+The `Shell` output:
+
+```Shell
+true
+```
+
+---
+
+## toInt() Method
+
+The `toInt()` method converts a value to an integer.
+
+## toInt Method Example
+
+```dart
+void main() {
+  double doubleValue = 42.75;
+  int intValue = doubleValue.toInt();
+
+  var checkCondition = (intValue == 42) ? true : false;
+
+  print(checkCondition);
+}
+```
+
+The `Shell` output:
+
+```Shell
+true
+```
+
+---
+
+## parse() Method
+
+The `parse()` method converts a String to a numeric type. Parses a string containing a number literal into a number.
+
+### parse() Method Variations
+
+### `num.parse()`
+
+  The method first tries to read the input as `integer` (similar to `int.parse` without a `radix`). If that fails, it tries to parse the input as a `double` (similar to `double.parse`). If that fails, too, it throws a `FormatException`.
+
+### `int.parse()`
+
+  Converts a String to an `integer` type value. If that fails, too, it throws a `FormatException`.
+
+### `double.parse()`
+
+  Converts a String to an `double` type value. If that fails, too, it throws a `FormatException`.
+
+## parse Method Example
+
+```dart
+void main() {
+  String stringValue = "42";
+  int intValue = int.parse(stringValue);
+
+  var checkCondition = (intValue.runtimeType == int) ? true : false;
+  print(checkCondition);
+}
+```
+
+The `Shell` output:
+
+```Shell
+true
+```
+
+---
+
```

Comment:
We don't need any specific function explanation or example in this case. Please remove the above-mentioned lines.

Reply:
Done